cql3/statements/cf_prop_defs.cc

```diff
@@ -52,6 +54,8 @@ const sstring cf_prop_defs::COMPACTION_STRATEGY_CLASS_KEY = "class";
 
 const sstring cf_prop_defs::COMPACTION_ENABLED_KEY = "enabled";
 
+const sstring cf_prop_defs::KW_TABLET_HINTS = "tablet_hints";
```

Comment:
It can be "tablets =", and we can gather everything tablet related there.

Reply:
Can do